Sources/BraveWallet/Preview Content/MockContent.swift

```diff
@@ -45,6 +47,7 @@ extension BraveWallet.BlockchainToken {
     logo: "",
     isErc20: false,
     isErc721: false,
```

Comment:
its not this PR, and its just mock tokens. but DAI and USDC are both ERC20 standard tokens.